src/lib/web-component/log-view/log-view.css

```diff
@@ -0,0 +1,85 @@
+.activity-log-container {
+  position: relative;
+}
+
+.log-table-wrapper {
+  overflow-y: scroll;
+  max-height: 500px;
+  height: auto;
+}
+
+.close {
+  background: url(/icons/close.svg) no-repeat center center / cover;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  top: 5px;
+  right: 5px;
+  cursor: pointer;
+}
+
+#logDetails {
+  margin: 50px 15px 10px;
+}
+
+.width-60 {
+  width: 60%;
+}
+
+.log-detail-wrapper {
+  position: absolute;
+  top: 0;
+  height: 500px;
+  right: 0;
+  width: 40%;
+  word-wrap: anywhere;
+  background-color: #ffffff;
+  border: 2px solid #000000;
+}
+
+table {
+  width: 100%;
+  border: 1px solid #eeeeee;
+  border-collapse: collapse;
+  width: 100%;
+  cursor: pointer;
+}
+
+table thead {
+  position: sticky;
+  top: 0;
+}
+
+th {
+  width: 100%;
+  background: #000;
+  padding: 5px;
+  text-transform: uppercase;
+  color: #ffffff;
+  flex: 1 1 20%;
+  text-align: center;
+  word-wrap: anywhere;
+  align-self: center;
+}
+
+thead tr {
+  display: flex;
+  width: 100%;
+}
+
+tbody tr:nth-of-type(odd) {
+  background: #eeeeee;
+}
+
+tbody tr {
+  display: flex;
```

Comment:
Is it really necessary to change from `display: table-row` (default) to `display: flex`?

Reply:
Sorry forget to change this, done now. :-)